web/src/reactions.js

```diff
@@ -205,10 +205,8 @@ export function get_reaction_title_data(message_id, local_id) {
 }
 
 export function get_reaction_section(message_id) {
```

Comment:
Let's pluralize this to make clear it potentially returns multiple elements.

Reply:
Done.